api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceImpl.java

```diff
@@ -92,10 +92,13 @@ && haveCreditsExpired(user)
                   WorkspaceUtils.isFreeTier(
                       ws.getBillingAccountName(), workbenchConfigProvider.get()))
           .filter(DbWorkspace::isActive)
-          .filter(ws -> ws.getBillingStatus().equals(BillingStatus.ACTIVE))
+          .filter(ws -> !ws.isInitialCreditsExpired())
+          .filter(ws -> !ws.isInitialCreditsExhausted())
```

Comment:
do we want this check?  I'm thinking about what happens when the credits expire after they have already been exhausted.  I think we *do* want to set "expired" in that case.

Reply:
Hmmm, good point. 

Would we want to send the email in that case or just set expired?